CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # CHANGELOG
+0.2.50
+- fix: error when client_id absent
```

Comment:
You also have to update the version in pyproject.toml file

Reply:
Done